package.json

```diff
@@ -52,6 +52,7 @@
     "mocha": "^2.2.5",
     "mocha-jsdom": "^1.0.0",
     "react": "^0.14.0",
+    "prop-types": "^15.5.0",
```

Comment:
1. Shouldn't it be in `dependencies` instead?

2. It's not compatible with `"react": "^0.14.0 || ^15.0.0"`, but only with `"react": "^0.14.9 || ^15.3.0"`, so peerDependencies should be modified accordingly.

3. Examples are meant to be independent (with their dependencies), so we should add it there as well.